includes/class-wc-payments-order-service.php

```diff
@@ -1930,7 +1928,14 @@ private function complete_order_processing( $order, $intent_status = null ) {
 	 * @return string The formatted order total.
 	 */
 	private function get_order_amount( $order ) {
-		return WC_Payments_Explicit_Price_Formatter::get_explicit_price( wc_price( $order->get_total(), [ 'currency' => $order->get_currency() ] ), $order );
+		$multi_currency_instance = WC_Payments_Multi_Currency();
+		$order_price             = $order->get_total();
+		if ( ! $order_price ) {
+			return $order_price;
+		}
```

Comment:
To match the previous state of the code, we can remove this check, as the order total will always be passed through `wc_price`.

Reply:
Fair, removed.